### PR TITLE
pacman/pacman.cpp: Add a new The Glob clone on Pac-Man hardware

### PR DESCRIPTION
**Pull request by clawgrip**
New working clone
---------------------
The Glob (Pac-Man hardware, set 2) [Arcadedealer aka Edgar Solé aka Dios, Recreativas.org]

### CODE REVIEW

File: src/mame/pacman/pacman.cpp [hunk 1]
```diff
@@ -7341,7 +7340,7 @@ ROM_START( sprglobp )
 	ROM_LOAD( "82s126.3m"  ,  0x0100, 0x0100, CRC(77245b66) SHA1(0c4d0bee858b97632411c440bea6948a74759746) )    // Timing - not used
 ROM_END
 
-/* This set is from a modified Pengo board.  Pengo and Pacman are functionally the same.
+/* This set is from a modified Pengo board. Pengo and Pacman are functionally the same.
```
Comment (cuavas) [inline src/mame/pacman/pacman.cpp:7343]: Please leave two spaces separating sentences alone.

File: src/mame/pacman/pacman.cpp [hunk 2]
```diff
@@ -8582,7 +8609,7 @@ void pacman_state::init_mspackpls()
  *
  *************************************/
 
-//          rom       parent    machine   inp       state          init
+//          ROM       PARENT    MACHINE   INPUT     STATE          INIT
```
Comment (cuavas) [inline src/mame/pacman/pacman.cpp:8612]: Please don’t do this.  excessive capitals are like screaming, and it doesn’t make it easier to read.

File: src/mame/pacman/pacman.cpp [hunk 3]
```diff
@@ -7383,6 +7381,35 @@ ROM_START( theglobme )
 	ROM_LOAD( "63s141_b.3m",     0x0100, 0x0100, CRC(77245b66) SHA1(0c4d0bee858b97632411c440bea6948a74759746) )    // Timing - not used
 ROM_END
 
+ROM_START( theglobpa )
+	ROM_REGION( 0x20000, "maincpu", 0 )
+	ROM_LOAD( "1-2516.bin",   0x0000, 0x0800, CRC(760f4764) SHA1(fc29aa0a1d5da28a40590ef37bbd1255713fb1a4) )
+	ROM_LOAD( "2-2516.bin",   0x0800, 0x0800, CRC(7d556bc6) SHA1(aef2b6799bd2a51e3de8282361d0c03c5bedb8ad) )
+	ROM_LOAD( "3-2516.bin",   0x1000, 0x0800, CRC(ca9dafca) SHA1(41cb4313f9a46cb40c7b41ea8c7c522c1d4b5d54) )
+	ROM_LOAD( "4-2516.bin",   0x1800, 0x0800, CRC(fff64f47) SHA1(a9b8a5e1641626eff312b4588d8afae8f8811e6d) )
+	ROM_LOAD( "5-2716.bin",   0x2000, 0x0800, CRC(3c352e0f) SHA1(5bc30414da27a96f9e96e3dccccc0a2d66c92731) )
```
Comment (cuavas) [inline src/mame/pacman/pacman.cpp:7390]: What’s the reasoning for putting the new ROM definitions here?  It’s generally easier to navigate if the order matches the order of system definition macros.

Comment (clawgrip) [reply src/mame/pacman/pacman.cpp]: My bad. Was there until knowing the closest set...

File: src/mame/pacman/pacman.cpp (hunk 2 above)
Comment (clawgrip) [reply src/mame/pacman/pacman.cpp]: Reverted!

File: src/mame/pacman/pacman.cpp (hunk 3 above)
Comment (clawgrip) [reply src/mame/pacman/pacman.cpp]: It's fixed now

**Pull request merged**